Katello/katello
# Fixes #26243 - remove file content

This PR introduces support for removing file repository content (backed by pulp3 servers).

To test this PR:
- Create a file repository with an upstream URL of `https://repos.fedorapeople.org/repos/pulp/pulp/fixtures/file/`
- Sync the repository.
- Navigate to the repository details  at `/products/n/repositories/y`and then to the content files page. Observe that under the repository details thee are 3 files, "1.iso", "2.iso", "3.iso".
- You can also confirm the file contents from the console, for example
`Katello::Repository.find(y).repository_files`.  Note also the repository version_href .
- Select "1.iso" and click the "Remove Files" button.
- After the task completes, the repository should only have content files "2.iso" and "3.iso" associated.
- From the console you can verify that the repository version_href has been updated.
